### PR TITLE
Pasha/nov15(pasha)

### PR DESCRIPTION
**Pull request by Pahan4k**
US1  - Bug, failed; US2 - ready

### CODE REVIEW

File: src/test/java/com/Briteerp/Pages/LunchPage.java [hunk 1]
```diff
@@ -1,4 +1,97 @@
...
+
+
+
+//---------------------------US1 WebElements & Methods------------------------------------------------------------------
+@FindBy(xpath = "//a[@data-menu-xmlid='lunch.lunch_order_menu_form']")
+public WebElement NewOrderBut;
+
+
+
+//**********************************************************************************************************************
```
Comment (Mosavi1988) [inline src/test/java/com/Briteerp/Pages/LunchPage.java]: Looks good

File: src/test/java/com/Briteerp/Pages/LunchPage.java [hunk 2]
```diff
@@ -1,4 +1,97 @@
...
+
+        }else  if (tabs.trim().equalsIgnoreCase("Employee Payments") && expectResult.trim()   .equalsIgnoreCase("Register Cash Moves")){
+
+        }
+
+
+
+    }
+
+
```
Comment (Mosavi1988) [inline src/test/java/com/Briteerp/Pages/LunchPage.java]: awesome

File: src/test/java/com/Briteerp/Runner/Runner1.java [hunk 3]
```diff
@@ -12,7 +12,7 @@
             },
             features = "src/test/resources/Features",
             glue = "com/Briteerp/StepDefinitions",
-            dryRun = true,
+            dryRun = false,
             //tags = "",
             publish = true
     )
```
Comment (Mosavi1988) [inline src/test/java/com/Briteerp/Runner/Runner1.java:18]: minor change but no worries

File: src/test/java/com/Briteerp/StepDefinitions/Lunch/Lunch_POM.java [hunk 4]
```diff
@@ -1,4 +1,174 @@
...
+
+    }
+    @Then("check expenses by clicking on {string} tab")
+    public void check_expenses_by_clicking_on_tab(String string) {
+
+    }
+////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+    //                                                US2
+
+//////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
```
Comment (Mosavi1988) [inline src/test/java/com/Briteerp/StepDefinitions/Lunch/Lunch_POM.java:65]: good job

File: src/test/java/com/Briteerp/StepDefinitions/Lunch/Lunch_POM.java [hunk 5]
```diff
@@ -1,4 +1,174 @@
...
+    }
+    @Then("user should be able to see Products page")
+    public void user_should_be_able_to_see_products_page() {
+        // Write code here that turns the phrase above into concrete actions
+        throw new io.cucumber.java.PendingException();
+    }
+    @Then("create new product")
+    public void create_new_product() {
+        // Write code here that turns the phrase above into concrete actions
+        throw new io.cucumber.java.PendingException();
```
Comment (Mosavi1988) [inline src/test/java/com/Briteerp/StepDefinitions/Lunch/Lunch_POM.java]: looks pretty good

File: src/test/java/com/Briteerp/StepDefinitions/Lunch/Lunch_POM.java [hunk 6]
```diff
@@ -1,4 +1,174 @@
...
+    // Write code here that turns the phrase above into concrete actions
+    throw new io.cucumber.java.PendingException();
+}
+    @Then("user should be able to see Products Categories page")
+    public void user_should_be_able_to_see_products_categories_page() {
+        // Write code here that turns the phrase above into concrete actions
+        throw new io.cucumber.java.PendingException();
+    }
+    @Then("create new product categories")
+    public void create_new_product_categories() {
```
Comment (Mosavi1988) [inline src/test/java/com/Briteerp/StepDefinitions/Lunch/Lunch_POM.java:218]: Scenarios looks good to me

File: src/test/java/com/Briteerp/StepDefinitions/Lunch/Lunch_POM.java [hunk 7]
```diff
@@ -1,4 +1,174 @@
...
+//////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+    @When("user clicking on {string} tab")
+    public void user_click_on_Alert_tab(String string) {
+        // Write code here that turns the phrase above into concrete actions
+        throw new io.cucumber.java.PendingException();
+    }
+    @Then("user should be able to see Alerts page")
+    public void user_should_be_able_to_see_alerts_page() {
+        // Write code here that turns the phrase above into concrete actions
```
Comment (Mosavi1988) [inline src/test/java/com/Briteerp/StepDefinitions/Lunch/Lunch_POM.java]: good job

**Review by Mosavi1988 — APPROVED** [review]

**Review by Mosavi1988 — APPROVED** [review]

**Review by Mosavi1988 — APPROVED** [review]

**Review by Mosavi1988 — APPROVED** [review]

**Review by Mosavi1988 — APPROVED** [review]

**Review by Mosavi1988 — APPROVED** [review]

File: src/test/java/com/Briteerp/Pages/LunchPage.java (hunk 1 above)
Comment (Pahan4k) [reply src/test/java/com/Briteerp/Pages/LunchPage.java]: Thank you Sayed

**Review by Mosavi1988 — APPROVED** [review]

**Review by numuminov — APPROVED** [review]

**Pull request merged**